rest-server/engines/src/main/scala/com/actionml/engines/urnavhinting/URNavHintingPreparator.scala

```diff
@@ -137,7 +137,7 @@ object URNavHintingPreparator extends LazyLogging with SparkMongoSupport {
 }
 
 /** Companion Object to construct an IndexedDatasetSpark from String Pair RDDs */
-object IndexedDatasetSpark {
+object IndexedDatasetSparkHelper {
```

Comment:
Same comment as above.

Reply:
There is an import of IndexedDatasetSpark from mahout. And at this line, you hide it with this declaration. This situation confuses me and the compiler (there is a warning). So I think it's better to have separate utility object for creating IndexedDatasetSpark instances.